src/main/java/http/configure/ConfigurationManager.java

```diff
@@ -0,0 +1,38 @@
+package http.configure;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.lang.module.Configuration;
+
+public class ConfigurationManager {
+    private static ConfigurationManager configurationManager;
+    private static Configuration currentConfiguration;
+
+    private ConfigurationManager() {
+
+    }
+
+    public static ConfigurationManager getInstance() {
```

Comment:
Nice use of the singleton pattern 🙂 

This works just fine here, but if you're interested you could have a look at [this article](https://www.journaldev.com/171/thread-safety-in-java-singleton-classes) explaining a potential flaw with the singleton pattern and how to overcome it.

Reply:
Thanks Sam this is outdated, I've removed this code.

Reply:
I'll give that article a read incase I run into this again